### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by valeriahhdez**
### Describe your changes
This commit contains text edits, describes how to solve spell-check failures, and adds statements and links about signing the CLA, verifying the commit signature, and displaying the 'Verified' status.

### Checklist before opening this PR (put `x` in the checkboxes)
- [ x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by valeriahhdez** [comment]
No worries Catherine @CathPag, take your time

File: content/en/contribute/_index.md [hunk 1]
```diff
@@ -7,111 +7,127 @@ menu:
...
+
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space.
```
Comment (CathPag) [inline content/en/contribute/_index.md:29]:
```suggestion
How you can participate in this project depends on your level of cloud native expertise. 
```
I really want to stay away from "anyone can contribute." First of all, that is not true. We've seen a bunch of contributions from people who do not have the qualification to do so. While localization is an option, it isn't for people who only are proficient in English. 

We introduced this section because of the false belief that everyone can participate. Without the proper know-how, it creates a lot of frustration for contributors and maintainers. I want us to be very transparent here to avoid that. Currently, I'm pointing everyone who wants to contribute a new term to this section and have them confirm that they qualify. It may sound harsh, but we need to set clear expectations.

File: content/en/contribute/_index.md [hunk 2]
```diff
@@ -7,111 +7,127 @@ menu:
...
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space.
```
Comment (CathPag) [inline content/en/contribute/_index.md:30]:
```suggestion
Simplifying complex concepts requires a deep knowledge of the topic. 
Therefore, to contribute new terms, you must be proficient in them. 
Contributors are typically engineers who have worked with these technologies for some time or academics focused on cloud native. 

That know-how is required because explaining complex concepts in simple words is _really_ hard. And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity results from hard work and collaboration between cloud native experts. 
```

Moving that part up again to further explain that, just because the result looks easy, it meant it is easy.

File: content/en/contribute/_index.md [hunk 3]
```diff
@@ -7,111 +7,127 @@ menu:
...
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space. 
+
+The localization effort is where beginners can make valuable contributions to the glossary.
```
Comment (CathPag) [inline content/en/contribute/_index.md:32]:
```suggestion
The localization effort is where beginners proficient in another language can make valuable contributions to the Glossary. 
```

File: content/en/contribute/_index.md [hunk 4]
```diff
@@ -7,111 +7,127 @@ menu:
...
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space. 
+
+The localization effort is where beginners can make valuable contributions to the glossary. 
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read the section [Help translate the glossary](#help-translate-the-glossary) of this guide to learn how to get started.
```
Comment (CathPag) [inline content/en/contribute/_index.md:33]:
```suggestion
With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read this guide's [Help Localize the glossary](#help-localize-the-glossary) section to learn how to get started. 
```

File: content/en/contribute/_index.md [hunk 5]
```diff
@@ -7,111 +7,127 @@ menu:
...
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space. 
+
+The localization effort is where beginners can make valuable contributions to the glossary. 
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read the section [Help translate the glossary](#help-translate-the-glossary) of this guide to learn how to get started. 
+
+## Before you start
+
+Before making any contribution to the glossary, please make sure of the following:
```
Comment (CathPag) [inline content/en/contribute/_index.md:37]:
```suggestion
Before beginning your Glossary contributor journey, be sure to:
```

File: content/en/contribute/_index.md [hunk 6]
```diff
@@ -7,111 +7,127 @@ menu:
...
+
+1. If you don't already have one, create a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account).
```
Comment (CathPag) [inline content/en/contribute/_index.md:39]:
```suggestion
1. Create a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account), if you don't have one already. 
```

File: content/en/contribute/_index.md [hunk 7]
```diff
@@ -7,111 +7,127 @@ menu:
...
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space. 
+
+The localization effort is where beginners can make valuable contributions to the glossary. 
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read the section [Help translate the glossary](#help-translate-the-glossary) of this guide to learn how to get started. 
+
+## Before you start
```
Comment (CathPag) [inline content/en/contribute/_index.md:35]: Not sure how to do this, but external links should open in a new tab. @seokho-son @iamNoah1, can you help?

File: content/en/contribute/_index.md [hunk 8]
```diff
@@ -7,111 +7,127 @@ menu:
...
+2. Sign the [Contributor License Agreement](https://docs.linuxfoundation.org/lfx/easycla/v2-current/contributors) (CLA). 
+3. [Verify your commit signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)
+4. Enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "Verified" status on your commits. 
+
+## Best practices {#best-practices}
 
-Explaining complex concepts in simple words is *really* hard. 
-And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity is the result of hard work and collaboration between cloud native experts.
-In short, to contribute new terms, you need a deep understanding of the topic. 
+To facilitate the reviewing process, our project uses [semantic line breaks](https://sembr.org/) (e.g., one line per sentence).
```
Comment (CathPag) [inline content/en/contribute/_index.md:46]:
```suggestion
To facilitate the reviewing process, please use [semantic line breaks](https://sembr.org/) (e.g., one line per sentence).
```

File: content/en/contribute/_index.md [hunk 9]
```diff
@@ -7,111 +7,127 @@ menu:
...
+We recommend checking out this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/) 
+to correctly format Markdown text in GitHub (e.g., hyperlink, bold, italic).
+To name the .md file of each term we use lowercase letters, hyphens instead of spaces to separate words, and we don't use parenthesis.
```
Comment (CathPag) [inline content/en/contribute/_index.md:49]:
```suggestion
And when naming .md files, please use lowercase letters and hyphens instead of spaces to separate words and avoid parenthesis.
```

File: content/en/contribute/_index.md [hunk 10]
```diff
@@ -7,111 +7,127 @@ menu:
...
+You can find the meeting details in the [CNCF calendar](https://www.cncf.io/calendar/). 
 You can also connect with the maintainers and fellow contributors in our [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel on the CNCF Slack 
 — we'd love to meet you! 
 
 ## Work on an existing issue {#work-on-an-existing-issue}
 
-Go to the [Glossary GitHub repo issues](https://github.com/cncf/glossary/issues). 
-There you'll see a list of all issues. You can filter by label (e.g. English language, help needed, good first issue). 
-Note that you'll need a GitHub account to do any of this.
+Go to the [Glossary GitHub repo issues](https://github.com/cncf/glossary/issues) where you will find a list of available issues. You can use labels (e.g., English language, help needed, good first issue) to filter out issues.
```
Comment (CathPag) [inline content/en/contribute/_index.md:64]:
```suggestion
Go to the [Glossary GitHub repo issues](https://github.com/cncf/glossary/issues) to find a list of available issues. 
You can use labels (e.g., English language, help needed, good first issue) to filter out issues.
```

File: content/en/contribute/_index.md [hunk 11]
```diff
@@ -7,111 +7,127 @@ menu:
...
 
-Make sure the term you are interested in is not already assigned to someone. 
-Here you can see that the first three terms are available while the next term has already been assigned.
+Be sure the term you select hasn't already been assigned to anybody. 
+For example, here you can see that the first three terms are available while the fourth one has already been assigned.
 
 ![assigning a term](/images/how-to/howto-04.png)
 
-Once you found a term you want to work on, say so in the issue. Click on it and add a comment.
+Once you select a term to work on, write a comment in the issue.
```
Comment (CathPag) [inline content/en/contribute/_index.md:73]:
```suggestion
Once you select a term to work on, comment on the issue. 
```

File: content/en/contribute/_index.md [hunk 12]
```diff
@@ -7,111 +7,127 @@ menu:
...
 
 ![Claiming an issue](/images/how-to/claiming-an-issue.png)
 
-Additionally, please also join the [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel on the CNCF Slack and 
-let the maintainers know that you've raised an issue for a new term 
-(ideally, tag _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ to be sure they don't miss it). 
-Note that you can only claim one term at a time. 
-If you want to work on multiple terms, please finish one before claiming the next one.
+Additionally, notify the maintainers on the [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel of the CNCF Slack workspace and 
+tag _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ to be sure they don't miss it.
```
Comment (CathPag) [inline content/en/contribute/_index.md:78]:
```suggestion
tag _@Catherine Paganini_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ to be sure they don't miss it. 
```
Removing Jason :)

**Review by CathPag — COMMENTED** [review]
Thank you! I got started. A few minor edits + more emphasis on who can contribute. Why is explained in the comments.

File: content/en/contribute/_index.md (hunk 5 above)
Comment (CathPag) [reply content/en/contribute/_index.md:42]: Ok :)

File: content/en/contribute/_index.md [hunk 13]
```diff
@@ -7,111 +7,133 @@ menu:
...
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+How you can participate in this project depends on your level of cloud native expertise. 
+Simplifying complex concepts requires a deep knowledge of the topic. 
+Therefore, to contribute new terms, you must be proficient in them. 
+Contributors are typically engineers who have worked with these technologies for some time or academics focused on cloud native. 
+
+That know-how is required because explaining complex concepts in simple words is _really_ hard. And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity results from hard work and collaboration between cloud native experts. 
+
```
Comment (CathPag) [inline content/en/contribute/_index.md:36]:
```suggestion

If you are not a cloud native expert yet but still want to contribute, we recommend teaming up with someone who is. 
Once the expert is confident that the term accurately describes the concept, you can submit a PR.
```

File: content/en/contribute/_index.md (hunk 7 above)
Comment (seokho-son) [reply content/en/contribute/_index.md:42]: 'shift + click on the link' ? :)

Comment (cjyabraham) [reply content/en/contribute/_index.md:42]: FYI we no longer force people to open external links in new tabs. I wrote up some reasons for that which I’ll copy below:
As part of the accessibility improvements we’ve been working on for our websites, we’d like to get away from forcing external sites to open in new browser tabs. The idea behind this was originally to prevent people from leaving our site or at least giving them an easy way to get back to our site by leaving the tab open. This practice, however, is not recommended for a few reasons:
- It can be disorienting to screen reader users, those with cognitive impairments or people who magnify their screen
- In a lot of cases it actually makes it harder to go back to the page you left since the Back button won’t take you there. This affects all users and is exacerbated on mobile where the old window is never visible.
- Opening a new tab takes control out of the user’s hands
[See this article for a full analysis](https://www.nngroup.com/articles/new-browser-windows-and-tabs/).

**Review by CathPag — APPROVED** [review]
Looks great, thank you!!

File: content/en/contribute/_index.md (hunk 7 above)
Comment (jihoon-seo) [reply content/en/contribute/_index.md:43]: Just FYI:
we cannot achieve this (make the link open in new tab) with markdown, so we have to use HTML if we want to.

Ref: https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown

File: content/en/contribute/_index.md [hunk 14]
```diff
@@ -7,111 +7,136 @@ menu:
...
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read this guide's [Help Localize the glossary](#help-localize-the-glossary) section to learn how to get started. 
+
+## Before you start
+
+Before beginning your Glossary contributor journey, be sure to complete the following steps:
+
+1. Create a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account), if you don't have one already. 
+2. Sign the [Contributor License Agreement](https://docs.linuxfoundation.org/lfx/easycla/v2-current/contributors) (CLA). 
+3. [Verify your commit signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)
+4. Enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "Verified" status on your commits.
```
Comment (seokho-son) [inline content/en/contribute/_index.md:50]: Hi @CathPag @valeriahhdez 
I'm not sure step 3 is an actionable step at this early stage. I believe that commit signature verification can only be done after a contributor has pushed a commit (or opened a PR) to GitHub repository. I think we can skip the step 3 and utilize the link only.

```suggestion
3. Enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "[Verified](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)" status on your commits. 
```

Comment (valeriahhdez) [reply content/en/contribute/_index.md:50]: The link I provided in this step is to verify commit signatures with SSH or GPG keys, which can be completed using the GitHub interface before making any commit or push. Maybe you're referring to the -S flag used to verify with the git commit command?
We could add that option too and make it clear that signing commits this way is only required at the moment of committing the changes.

Comment (seokho-son) [reply content/en/contribute/_index.md:50]: Hi @valeriahhdez 
Thank you for the clarification. I misunderstood. I thought the step is to check if a commit contains a verified signature. 
I think we can merge step 3, 4 into one since they are very related subjects (related with commit signature). It is to simplify the steps.

Applying my comment is up to you.
LGTM as is :) 

```suggestion
3. [Configure your commit signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) and enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "Verified" status on your commits. 
```

**Review by seokho-son — APPROVED** [review]

**Review by krol3 — APPROVED** [review]
lgtm! Awesome contribution @valeriahhdez
